### PR TITLE
Add recipe for lsd

### PR DESCRIPTION
**Pull request by m-rossi**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by m-rossi** [comment]
@conda-forge/bioconda-recipes looks like the `lsd` package is something completely different. I think we have two possibilities here:
- Keep the recipe as it is
- Rename the package from `lsd` to `lsdeluxe`

**Comment by m-rossi** [comment]
I went with the second option.

@conda-forge/staged-recipes this recipe is ready for review.

**Comment by jakirkham** [comment]
We need to bundle any dependency licenses. AIUI `cargo-bundle-licenses` does this. For [example]( https://github.com/conda-forge/vegafusion-python-embed-feedstock/blob/305c0d884ef868257f12ddf7280690668a2c0a86/recipe/meta.yaml#L16 ), which needs to be a [`build` dependency]( https://github.com/conda-forge/vegafusion-python-embed-feedstock/blob/305c0d884ef868257f12ddf7280690668a2c0a86/recipe/meta.yaml#L24 )

**Comment by BastianZim** [comment]
> May we also add this somewhere to the conda-forge docs?

The maintainer of the package wanted to add some more info to their readme first and then I was planning on adding it but it seems like so many use it now, that it makes sense to include it, I'd say. Do you want to make the PR?

Ref https://github.com/sstadick/cargo-bundle-licenses/issues/4#issuecomment-993797362

**Comment by m-rossi** [comment]
> The maintainer of the package wanted to add some more info to their readme first and then I was planning on adding it but it seems like so many use it now, that it makes sense to include it, I'd say. Do you want to make the PR?

@BastianZim Since you started the discussion with the creator, I'll let you go first. If you want me to help or create it instead, let me know.

**Comment by m-rossi** [comment]
@conda-forge/staged-recipes I think its time for another review.

**Comment by carterbox** [comment]
@conda-forge-admin, please restart ci

File: recipes/lsdeluxe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Peltoche/lsd/archive/{{ version }}.tar.gz
+  sha256: f500c18221f9c3fd45f88f6f764001e99cf9d6d74af9172cbb9a9ff32f3e5c7d
+
+build:
+  number: 0
```
Comment (carterbox) [inline recipes/lsdeluxe/meta.yaml:13]:
```suggestion
build:
  number: 0
  ignore_run_exports_from:
    - {{ compiler('cxx') }}
```
Overdepending according to link inspector. Probably binary doesn't use CXX standard library.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=497376&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1150

**Review by carterbox — APPROVED** [review]
Merge if all build passing.

File: recipes/lsdeluxe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  ignore_run_exports_from:
+    - {{ compiler('cxx') }}
```
Comment (carterbox) [inline recipes/lsdeluxe/meta.yaml:15]:
```suggestion
  ignore_run_exports:
    - libstdcxx-ng

Comment (isuruf) [reply recipes/lsdeluxe/meta.yaml]: Why?

**Pull request merged**